libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -112,6 +112,7 @@ import { IWidgetDefinition } from '@gooddata/sdk-backend-spi';
 import { IWorkspacePermissions } from '@gooddata/sdk-backend-spi';
 import { LocalIdRef } from '@gooddata/sdk-model';
 import { MemoizedFunction } from 'lodash';
+import { MouseEvent as MouseEvent_2 } from 'react';
```

Comment:
remove this renaming to MouseEvent_2

Reply:
this was automatically generated by the tooling :-)